### PR TITLE
Add WebSocketStream

### PR DESCRIPTION
**Pull request by Elchi3**
Discovered by the @openwebdocs BCD collector thanks to https://github.com/openwebdocs/mdn-bcd-collector/pull/1341.
No spec_url yet, see https://github.com/whatwg/websockets/pull/48

Adding the "websocket-stream" tag, sounds good Philip?

### CODE REVIEW

**Comment by foolip** [comment]
> Adding the "websocket-stream" tag, sounds good Philip?

This is an interesting case. Because of https://caniuse.com/websockets and https://websockets.spec.whatwg.org/ I would say that "websockets" is the best name for the base feature which goes back ages. But if we follow the style of turning `CoolThing` APIs into "cool-thing" identifiers, this should be "web-socket-stream". That looks weird alongside "websockets", but maybe that's OK.

We can always rename it later, but for now I guess I'd go with "web-socket-stream" based on the `WebSocketStream` API name. It helps that this is distinct from the old `WebSocket` interface, and not a `stream` property or something on `WebSocket`.

File: api/WebSocketError.json [hunk 1]
```diff
@@ -0,0 +1,145 @@
+{
+  "api": {
+    "WebSocketError": {
+      "__compat": {
+        "tags": [
+          "web-features:websocket-stream"
```
Comment (foolip) [inline api/WebSocketError.json:6]: Suggesting for the first instance only:

```suggestion
          "web-features:web-socket-stream"
```

**Review by foolip — APPROVED** [review]

**Comment by Elchi3** [comment]
Thanks @foolip, I have no strong feelings about the id and accepted your suggestion. Merging this.

Do you want me to open a PR on web-features or do we have tooling in place that will signal to web-features that we have a new tag in BCD which will spawn something (automatically) over there?

**Comment by foolip** [comment]
@Elchi3 we don't have any tooling for this, but that would make a lot of sense. We could have a test that fails if there are any features in BCD that aren't in web-features, and a list for allowed exceptions which we could treat like a burndown list.

**Pull request merged**